backend/services/create_team.py

```diff
@@ -0,0 +1,52 @@
+from battle.battles.battle import validate_sum_pokemons, verify_pokemon_is_saved
```

Comment:
The name of this folder is confusing. My advice for naming services is to use the name of the app they are in. In this case, `services/battle.py`

Reply:
thanks :)